Projects/hCoreUI/Sources/GradientView.swift

```diff
@@ -78,7 +78,7 @@ extension GradientView: Viewable {
         bag += combineLatest(
             $shouldShowGradient.atOnce(),
             gradientView.traitCollectionSignal.atOnce(),
-            gradientView.didLayoutSignal.providedSignal.readable()
+            gradientView.signal(for: \.bounds).delay(by: 0.1).atOnce()
```

Comment:
Oh wow this is cool. I didn't realise you can observe properties like this!

Reply:
@tarikstafford yeah its dope, a caveat it only support `kvo`-observable properties tho